controllers/nstemplateset/namespaces_test.go

```diff
@@ -1024,10 +1120,14 @@ func TestUpdateNamespaces(t *testing.T) {
 
 		t.Run("update from abcde11 revision to abcde12 revision as part of the advanced tier", func(t *testing.T) {
 			// given
-			nsTmplSet := newNSTmplSet(namespaceName, spacename, "advanced", withNamespaces("abcde12", "dev"))
+			nsTmplSet := newNSTmplSet(namespaceName, spacename, "advanced", withNamespaces("abcde12", "dev"), withNSTemplateSetFeatureAnnotation("feature-1"))
 			devNS := newNamespace("advanced", spacename, "dev", withTemplateRefUsingRevision("abcde11"))
 			ro := newRole(devNS.Name, "exec-pods", spacename)
 			rb := newRoleBinding(devNS.Name, "crtadmin-pods", spacename)
+			featuredRB := newRoleBinding(devNS.Name, "feature-1-rb", spacename) // existing featured RB we need to make sure gets deleted after updating since the new template doesn't have this featured object
+			featuredRB.Annotations = map[string]string{
+				toolchainv1alpha1.FeatureToggleNameAnnotationKey: "feature-1-rb",
+			}
```

Comment:
shouldn't this be passed to the `prepareNamespacesManager` function or am I missing something ?

Reply:
We should! Thank you: https://github.com/codeready-toolchain/member-operator/pull/580/commits/4dbe286e0f90e4541e10878c21c7d94aba501db5
Otherwise the test doesn't really test what it's supposed to.